CMakeLists.txt

```diff
@@ -487,16 +487,36 @@ set(KERNELS_SOURCE_DIR ${PROJECT_SOURCE_DIR}/src/kernels)
 
 find_program(UNZIPPER lbzcat bzcat REQUIRED)
 file(MAKE_DIRECTORY ${KERNELS_BINARY_DIR})
+add_subdirectory(tools/sqlite2txt)
```

Comment:
It this tool always required? Can we make it optional?

Reply:
Made it only included when MIOPEN_ENABLE_SQLITE_PERFDB not enabled (as it wouldn't be needed otherwise).